ros2pkg/ros2pkg/resource/ament_python/main.py.em

```diff
@@ -0,0 +1,7 @@
+import rclpy
```

Comment:
these imports should maybe be considered from the dependencies, but I think it's not required to have `rclpy` every time.

Reply:
Removed rclpy in the last commit. This makes it more consistent with the ament_cmake template.